Detections/SecurityEvent/NonDCActiveDirectoryReplication.yaml

```diff
@@ -0,0 +1,56 @@
+id: b9d2eebc-5dcb-4888-8165-900db44443ab
+name: Non Domain Controller Active Directory Replication
+description: |
+  'This query uses Windows Event ID 4662 in order to detect potential attempts to synz an active directory object leveraging directory replication services (DRS)
+  from a non-domain-controller account with privileged permissions to do so. Ref: https://threathunterplaybook.com/notebooks/windows/06_credential_access/WIN-180815210510.html'
```

Comment:
Reads a bit odd in general.  Maybe say "detects potential attempts by accounts not associated with a domain controller to sync an AD object leveraging AD DRS".

Reply:
fixed